app/uk/gov/hmrc/serviceconfigs/service/BuildJobService.scala

```diff
@@ -45,8 +45,8 @@ class BuildJobService @Inject()(
       regex    = """jobs/live/(.*).groovy""".r
       blob     = "https://github.com/hmrc/build-jobs/blob"
       items    = ZipUtil
-                   .findRepos(zip, repos, regex, blob)
-                   .map { case (repo, location) => BuildJob(serviceName = ServiceName(repo.name), location = location) }
+                   .findRepos(zip, repos.map(_.repoName.asString), regex, blob)
+                   .map { case (name, location) => BuildJob(serviceName = ServiceName(name  ), location = location) }
```

Comment:
maybe move `asString` into `ServiceName(repoName.asString)` to keep to conversions clearer? Could continue to match on `repo`?
Same for `DashboardService`